src/modules/apis/likes.js

```diff
@@ -0,0 +1,20 @@
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/TpSQYZhyRfL6B96XWZQ9/likes/';
+
+export const addLike = async (id) => {
+  await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  });
+};
+
+export const catchLike = async () => {
+  const response = await fetch(url);
+  const resultJSON = await response.json();
+  const resultMessage = resultJSON;
+  console.log(resultMessage);
+};
```

Comment:
well, instead of creating `api.js` file to store the APIs you created another folder named `apis` and stored `like.js` into it. You were only supposed to create a new file in modules folder named as `api.js` to store the APIs into a different file